deployments/jwebbinar/image/environments/jwebbinar/tests/notebooks

```diff
@@ -0,0 +1,19 @@
+/opt/notebooks/notebooks/MAST/Kepler/Kepler_TPF/kepler_tpf.ipynb
```

Comment:
I'm going to delete these in my follow-on PR.   Are there any replacements?

Reply:
Not that I'm aware of.

Reply:
OK,  status update:   first I deleted these.   Later Greg defined the pipeline notebook as belonging to JWebbinar,  so I redefined jwebbinar/tests/notebooks to specify that one notebook as a test,  removing it from jdaviz.